policyengine_us/variables/household/income/person/pensions/ira_distributions.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_us.model_api import *
+
+
+class ira_distributions(Variable):
```

Comment:
i think we want to separate roth from traditional ira

could you check against sources in the asec codebook mentioned in https://github.com/PolicyEngine/policyengine-us/issues/3079?